homepage/homepage/content/docs/groups/sharing.mdx

```diff
@@ -85,8 +85,25 @@ useAcceptInvite({
   },
 });
 ```
-</CodeGroup>
 
+You can accept an invitation programmatically by using the `acceptInvite` method on an account.
+Pass the ID of the CoValue you're being invited to, the secret from the invite link, and the schema of the CoValue.
+
+<CodeGroup>
+```ts twoslash
+import { co, z } from "jazz-tools";
+
+const Organization = co.map({
+  name: z.string(),
+});
+
+await account.acceptInvite(
+  organizationId,
+  inviteSecret,
+  Organization
+);
```

Comment:
Our examples are typechecked, so all the variables need to be declared.
Using `// ---cut---` to hide the hacks to make it work.

```suggestion
```ts twoslash
import { co, z, Account } from "jazz-tools";

const Organization = co.map({
  name: z.string(),
});
const account = {} as unknown as Account;
const organizationId = "";
const inviteSecret ="";

// ---cut---
await account.acceptInvite(
  organizationId,
  inviteSecret,
  Organization
);
```